x-pack/plugins/ml/public/application/overview/components/anomaly_detection_panel/actions.tsx

```diff
@@ -7,9 +7,10 @@
 import React, { FC } from 'react';
 import { EuiToolTip, EuiButtonEmpty } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
-// @ts-ignore no module file
-import { getLink } from '../../../jobs/jobs_list/components/job_actions/results';
 import { MlSummaryJobs } from '../../../../../common/types/anomaly_detection_jobs';
+import { useCreateADLinks } from '../../../components/custom_hooks/use_create_ad_links';
+
+// @ts-ignore no module file
```

Comment:
looks like this comment can go

Reply:
Removed here https://github.com/elastic/kibana/pull/76347/commits/1118b3e08345110bec102f86bfbd3bf46cce969c